core/assets/bundles/bundle_fr.properties

```
@@ -2033,99 +2033,99 @@ block.repair-turret.description = Répare en continu l'unité endommagée la plu
+block.turbine-condenser.description = Génère de l'énergie lorsqu'il est placé sur des évents. Produit une petite quantité d'eau.
+block.chemical-combustion-chamber.description = Génère de l'énergie à partir d'arkycite et d'ozone.
+block.pyrolysis-generator.description = Génère de grandes quantités d'énergie à partir d'arkycite et de scories. Produit de l'eau comme sous-produit.
+block.flux-reactor.description = Génère de grandes quantités d'énergie lorsqu'il est chauffé. Requiert du cyanogène comme stabilisant. La puissance de sortie et les besoins en cyanogène sont proportionnels à l'apport de chaleur.\nExplose s'il n'y a pas assez de cyanogène.
+block.neoplasia-reactor.description = Utilise de l'arkycite, de l'eau et du tissu phasé pour générer de grandes quantités d'énergie. Produit de la chaleur et du néoplasme dangereux comme sous-produit.\nExplose violemment si le néoplasme n'est pas retiré du réacteur via des conduits.
+block.build-tower.description = Reconstruit automatiquement les structures à portée et assiste les autres unités en construction.
+block.regen-projector.description = Répare lentement les structures alliées dans un périmètre carré. Nécessite de l'hydrogène.\nPeut utiliser du tissu phasé pour augmenter l'efficacité.
+block.reinforced-container.description = Stocke une petite quantité d'objets. Le contenu peut être récupéré via des déchargeurs. N'augmente pas la capacité de stockage du noyau.
+block.reinforced-vault.description = Stocke une grande quantité d'objets. Le contenu peut être récupéré via des déchargeurs. N'augmente pas la capacité de stockage du noyau.
+block.tank-fabricator.description = Construit des unités Stell. Les unités de sortie peuvent être utilisées directement, ou transférées dans des refabricateurs pour une amélioration.
+block.ship-fabricator.description = Construit des unités Elude. Les unités de sortie peuvent être utilisées directement, ou transférées dans des refabricateurs pour une amélioration.
+block.mech-fabricator.description = Construit des unités Merui. Les unités de sortie peuvent être utilisées directement, ou transférées dans des refabricateurs pour une amélioration.
+block.tank-assembler.description = Assemble de grands tanks à partir de blocs et d'unités. Le niveau de l'unité de sortie peut être augmenté en ajoutant des modules.
+block.ship-assembler.description = Assemble de grands navires à partir de blocs et d'unités. Le niveau de l'unité de sortie peut être augmenté en ajoutant des modules.
+block.mech-assembler.description = Assemble de grands mechs à partir de blocs et d'unités. Le niveau de l'unité de sortie peut être augmenté en ajoutant des modules.
+block.tank-refabricator.description = Améliore les tanks au deuxième niveau.
+block.ship-refabricator.description = Améliore les navires au deuxième niveau.
+block.mech-refabricator.description = Améliore les mechs au deuxième niveau.
+block.prime-refabricator.description = Améliore les unités saisies au troisième niveau.
```

```suggestion
block.prime-refabricator.description = Améliore les unités au troisième niveau.
```
G oublié ce petit truc.

---

ça serait pas mieux de changer "navires" en "vaisseaux"? et "mechs" en "mécha"

---

Il est vrai